about/faq.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "WPE's Frequently Asked Questions"
+tags: [about]
+data: { dateless: "true" }
+permalink: /about/faq.html
+---
+
+[[toc]]
+
+## What is the difference between WebPlatformForEmbedded's WPE and WPEWebKit?
+
+Historically the [WebPlatformForEmbedded flavor of
+WPE](https://github.com/WebPlatformForEmbedded/WPEWebKit) came first. This
+version includes several adaptations for the RDK platform, as well as different
```

Comment:
Would it be fair to say it was developed for RDK? I feel like it would make sense to include something here that said "Reference Design Kit (RDK), a <1 sentence what is that/for">